tests/rust/secrets-reqwest-hardcoded-auth-rust-test.yml

```
@@ -0,0 +1,36 @@
+id: secrets-reqwest-hardcoded-auth-rust
+valid:
+  - |
+    use reqwest::Client; 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)